exodus/reports/views.py

```diff
@@ -48,29 +48,21 @@ def get_reports(request, handle=None):
                 reports = reports.filter(application__handle=handle)
     except Report.DoesNotExist:
         raise Http404(_("reports do not exist"))
-
     reports_paged = _paginate(request, reports)
+
+    app_count = Application.objects.distinct('name', 'handle').count()
```

Comment:
I know that the number of applications was calculated this way before, but I'm wondering whether it's correct.
I would simply write `app_count = Application.objects.distinct('handle').count()` because a simple application can change its name but it should still count as one.

Reply:
I agree, fixed!